src/System.Resources.Reader/src/System/Resources/StreamExtensions.cs

```diff
@@ -31,10 +31,46 @@ public static bool TryReadInt32(this Stream stream, out int result)
             return true;
         }
 
+        public static uint ReadUInt32(this Stream stream)
+        {
+            var b0 = stream.ReadByte();
+            if (b0 < 0) throw new EndOfStreamException();
+            var b1 = stream.ReadByte();
+            if (b1 < 0) throw new EndOfStreamException();
+            var b2 = stream.ReadByte();
+            if (b2 < 0) throw new EndOfStreamException();
+            var b3 = stream.ReadByte();
+            if (b3 < 0) throw new EndOfStreamException();
+
+            uint result = (uint)b3;
+            result <<= 8;
+            result |= (uint)b2;
+            result <<= 8;
+            result |= (uint)b1;
+            result <<= 8;
+            result |= (uint)b0;
+
+            return result;
+        }
+
+        public static ushort ReadUInt16(this Stream stream)
+        {
+            var b0 = stream.ReadByte();
+            if (b0 < 0) throw new EndOfStreamException();
+            var b1 = stream.ReadByte();
+            if (b1 < 0) throw new EndOfStreamException();
+
+            ushort result = (ushort)b1;
+            result <<= 8;
+            result |= (ushort)b0;
+
+            return result;
+        }
```

Comment:
I don't see these two new methods being used.  Why are they being added in this PR?